.github/workflows/verify-gha-unit-tests-count.yml

```diff
@@ -9,20 +9,21 @@ on:
 jobs:
   collect-and-verify:
     if: (github.repository == 'edx/edx-platform-private') || (github.repository == 'openedx/edx-platform' && (startsWith(github.base_ref, 'open-release') == false))
-    runs-on: [ edx-platform-runner ]
+    runs-on: [ "ubuntu-20.04" ]
```

Comment:
💡 Will be great if we could centralize the ubuntu versions into a constants file or in some open-edx-setup-action

Reply:
Tried to setup env variable here https://github.com/openedx/edx-platform/actions/runs/9281219260/workflow but gha doesn't allow this syntax. 
https://docs.github.com/en/actions/using-workflows/reusing-workflows seems to be the way here for DRY it.